### PR TITLE
Updated manual for database change.

### PR DESCRIPTION
**Pull request by stijnhau**
Updated manual for database change.

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ojhaujjwal) [inline .project:1]: Wait! Wait! What does that `.project` file mean?

Comment (stijnhau) [inline .project:1]: Has something to do with the editor (Zend Studio(eclipse)) i work in.
Shouldn't have commited it realy.

**Comment by ojhaujjwal** [comment]
Executing SQL command is optional. I have updated the installation docs!

**Comment by stijnhau** [comment]
Oke.
This issue will be closed.

**Pull request closed without merging**